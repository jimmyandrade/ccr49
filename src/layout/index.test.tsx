import {create} from "react-test-renderer";
import {Layout} from "./index";

jest.mock("../components/AppBar", () => "AppBar");
jest.mock("../components/Footer", () => "Footer");

describe("<Layout /> component", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<Layout>Test children</Layout>);
        expect(testRenderer).toMatchSnapshot();
    })
});