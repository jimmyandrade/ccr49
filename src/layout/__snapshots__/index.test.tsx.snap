// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> component should match snapshot 1`] = `
Array [
  <div
    className="MuiBox-root MuiBox-root-1"
  >
    <AppBar
      title="Jornada UOU"
    />
  </div>,
  "Test children",
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <Footer
      title="Jornada UOU"
    />
  </div>,
]
`;
